examples/using-gatsby-with-json-yaml/content/client-data.json

```diff
@@ -0,0 +1,17 @@
+{
+    "title":"JSON used in the client with React and Gatsby",
```

Comment:
This whole example needs to be updated to remove all mentions of "client" so as not to be misleading since the work is being done at build time.